apps/experiments/views/experiment.py

```diff
@@ -64,7 +64,23 @@ def get_queryset(self):
         return query_set
 
 
-class CreateExperiment(CreateView):
+class ExperimentViewMixin:
+    def get_form(self):
+        form = super().get_form()
+        _apply_related_model_querysets(self.request.team, form)
+        _apply_voice_provider_alpine_attrs(form)
+        return form
+
+    def form_valid(self, form):
+        form.instance.team = self.request.team
+        form.instance.owner = self.request.user
```

Comment:
Not a blocker but this will change the 'owner' to whichever user is updating the experiment.

At some point we should probably just create a custom form class for experiments.

Reply:
Ah good catch. For the sake of running the "same code" as before, I made this update: https://github.com/dimagi/open-chat-studio/pull/132/commits/8a12501d57a0220ff25a494abf81581f995075a5